server/links.go

```diff
@@ -0,0 +1,41 @@
+package server
+
+import (
+	"errors"
+	"net/url"
+)
+
+type getExternalLinksResponse struct {
+	StatusFeed  *string      `json:"statusFeed,omitempty"` // Location of the a JSON Feed for client's Status page News Feed
+	CustomLinks []CustomLink `json:"custom,omitempty"`     // Any custom external links for client's User menu
+}
+
+// CustomLink is a handler that returns a custom link to be used in server's routes response, within ExternalLinks
+type CustomLink struct {
+	Name string `json:"name,omitempty"`
+	URL  string `json:"url,omitempty"`
```

Comment:
Cool -- good thinking. There isn't, and it's currently written to throw an error in that circumstances, so there should not be an `omitempty`.